docs/tools/MASTG-TOOL-0105.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Corellium
+platform: generic
+source: https://corellium.com
+---
+
+Corellium is a powerful iOS and Android device virtualization platform that provides a comprehensive suite of tools for security researchers, developers, and testers. It allows users to create and manage virtual devices, perform dynamic analysis, and test applications in a controlled environment.
+
+## Overview
+
+Corellium offers a cloud-based solution that enables users to run virtualized iOS and Android devices. These virtual devices are fully functional and can be used for various purposes, including security testing, app development, and research. Corellium provides a web-based interface for managing virtual devices, as well as APIs for automation and integration with other tools.
```

Comment:
```suggestion
Corellium offers a cloud-based solution that enables users to run virtualized iOS and Android devices. These virtual devices can be used for various purposes, including security testing, app development, and research. Corellium provides a web-based interface for managing virtual devices, as well as APIs for automation and integration with other tools.
```
They're not 100% functional unfortunatelly. Not having a camera or, more importantly, no App Store, limits the usage of iOS devices. There's also no data connection for example.